src/3d/CMakeLists.txt

```diff
@@ -165,7 +165,7 @@ IF(MSVC)
   SET_TARGET_PROPERTIES(qgis_3d PROPERTIES LINK_FLAGS "/FORCE:MULTIPLE")
 ENDIF(MSVC)
 
-SET(QGIS_3D_HDRS ${QGIS_3D_HDRS} ${CMAKE_CURRENT_BINARY_DIR}/qgis_3d.h)
+SET(QGIS_3D_HDRS ${QGIS_3D_HDRS} ${QGIS_3D_PRIVATE_HDRS} ${CMAKE_CURRENT_BINARY_DIR}/qgis_3d.h)
```

Comment:
Don't!!
Move the erroneous ones to the public header section instead.

Reply:
ok, done